features/step_definitions/map_steps.rb

```diff
@@ -89,14 +92,17 @@ def marker_json_for_org_names(*org_names)
 #TODO if this ever needs refactoring, factor it in with what's in
 # config/initializers/webmock.rb
 def stub_request_with_address(address, body = nil)
-  filename = "#{address.gsub(/\s/, '_')}.json"
-  filename = File.read "test/fixtures/#{filename}"
-  # Webmock shows URLs with '%20' standing for space, but uri_encode susbtitutes with '+'
-  # So let's fix
-  addr_in_uri = address.uri_encode.gsub(/\+/, "%20")
-  # Stub request, which URL matches Regex
-  stub_request(:get, /http:\/\/maps.googleapis.com\/maps\/api\/geocode\/json\?address=#{addr_in_uri}/).
-      to_return(status => 200, :body => body || filename, :headers => {})
+  VCR.use_cassette( "googleapis" ) do
```

Comment:
I'd say the reason this is not working is that by placing the VCR.use_cassette here, we are wrapping the one place that is already stubbed.

Looking at the cucumber output, I think we have two kinds of 3rd party API requests causing us trouble:

a) "Geocoder: HTTP request being made for http://maps.googleapis.com/maps/api/geocode/json?address=83+pinner+road%2C+&language=en&sensor=false"

b) "Request to unknown URL: http://maps.gstatic.com/maps-api-v3/api/js/19/20/common.js"

the former (i.e. geocoder requests) can be handled without VCR, using this existing stub_request_with_address method - although we might argue that since we've failed to use it so many times it would be better to replace it with VCR for feature tests at least.

For the latter I think we'll have to use a slightly different VCR approach as described here: https://www.relishapp.com/vcr/vcr/v/2-9-3/docs/test-frameworks/usage-with-cucumber

i) Use VCR.use_cassette in a step definition.
ii) Use a VCR.cucumber_tags block to tell VCR to use a cassette for a tagged scenario.

What you've got about is the first type of cucumber VCR integration.  I think we've got to use a VCR.cucumber_tags block as described in that documentation:

`t.tag  '@vcr', :use_scenario_name => true`